bin/serviceinfo.js

```diff
@@ -120,6 +119,73 @@ db.init(function(err) {
             });
         },
 
+        async next=>{
+            /* Get all service info */
+            db.Serviceinfo.find({}).exec((err,services)=>{
+                if(err) return next(err);
+                services.forEach(async(service)=>{
+                    let countData;
+                    const currentDate = new Date();
+                    if(!service.monthlyCounts || !service.monthlyCounts.length) {
```

Comment:
There is a way to combine these 2 giant if statements into one. :) Can you think of a way?